src/korean.js

```diff
@@ -78,7 +79,16 @@ class Korean {
         return futureConjugation.substring(0, futureConjugation.length - 3);
       }
       case 'truncated':
```

Comment:
Line 80 is not required right? `case 'truncated':`

Reply:
Yes, this "truncated" option will not show on the UI.  However, this is kept for existing library users who have been using the "truncated" option to get the "present truncated" output.  For example, many of the Llip's lessons depend on  "truncated" conjugation.